.github/workflows/validate-updatecli.yaml

```diff
@@ -0,0 +1,43 @@
+name: Validate updatecli policies
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Setup updatecli
+        uses: updatecli/updatecli-action@b0d4fd84ae72108930c9adb83bb4beba1acbaf9f # v2.79.0
+
+      - name: Validate
+        working-directory: updatecli
+        run: make test
```

Comment:
```suggestion
        run: make --directory=updatecli test
```

Reply:
this is taken care by the `working-directory` key on the step.